Formula/anka-scripts.rb

```diff
@@ -1,4 +1,4 @@
-class AnkaGettingStartedScripts < Formula
+class AnkaScripts < Formula
```

Comment:
Please merge latest changes from master :)

Reply:
@NorseGaud Ok cool… Although now this formula also includes scripts from the other repo, so should it still share the `GettingStarted` name?

Reply:
Good point -- What do you think about separating these Formula so we end up with `anka-scripts` and `anka-getting-started-scripts` ?

Reply:
@NorseGaud Could do… Do they need to be in separate repositories in the first place?

Reply:
Yep, the "scripts" are just helper scripts. Our getting started repo are helper scripts, but they're used by our CI and a bunch of user guides/docs so I want to keep it as lightweight as humanly possible 😅